src/Actions/ManagesContacts.php

```diff
@@ -170,4 +218,22 @@ public function removeTagFromContact(Contact $contact, Tag $tag)
 
         $this->delete("contactTags/{$contactTag->id}");
     }
+
+    /**
+     * Determines the contact ID.
+     *
+     * @param Contact|int|string $contact
+     */
+    protected function getContactId($contact)
```

Comment:
This function is not needed when you forces to use the Contact object in the changes you made

Reply:
I think this is a nice convenience function. If I have just an ID in my database I can pass this in. If I do as you suggest I have to do an API call which, in my opinion is unnecessary overhead.

Reply:
Ok. I wrote a comment on the PR it itself. It is OK to enter the $id to updateContact and $deleteContact, but it should always be the $id.